### DISCUSSION TITLE
Any method for alerts?

### DISCUSSION THREAD

**Discussion by AshminJayson** [vadimdemedes/ink#595] [General]
Hey, I wanted to try building a pomodoro timer app using ink and was wondering if there was any way in which I could trigger an alert like a windows notification programmatically?

Any snippets of code or help in regard to the same is hugely appreciated.

**Comment by vadimdemedes** [comment]
You can use https://github.com/mikaelbr/node-notifier.

**Reply by AshminJayson** [reply]
How would I be able to implement this if I'm not using a NodeJS backend ? As in the CLI tool is only has a frontend?

**Reply by vadimdemedes** [reply]
Could you explain more what is it that you're trying to implement?
